sorc/link_workflow.sh

```diff
@@ -295,10 +295,43 @@ fi
 
 # GDASApp
 if [[ -d "${script_dir}/gdas.cd" ]]; then
-  for gdasexe in fv3jedi_addincrement.x fv3jedi_diffstates.x fv3jedi_ensvariance.x fv3jedi_hofx.x \
-    fv3jedi_var.x fv3jedi_convertincrement.x fv3jedi_dirac.x fv3jedi_error_covariance_training.x \
-    fv3jedi_letkf.x fv3jedi_convertstate.x fv3jedi_eda.x fv3jedi_forecast.x fv3jedi_plot_field.x \
-    fv3jedi_data_checker.py fv3jedi_enshofx.x fv3jedi_hofx_nomodel.x fv3jedi_testdata_downloader.py; do
+  declare -a JEDI_EXE=("fv3jedi_addincrement.x" \
+                       "fv3jedi_diffstates.x" \
+                       "fv3jedi_ensvariance.x" \
+                       "fv3jedi_hofx.x" \
+                       "fv3jedi_var.x" \
+                       "fv3jedi_convertincrement.x" \
+                       "fv3jedi_dirac.x" \
+                       "fv3jedi_error_covariance_training.x" \
+                       "fv3jedi_letkf.x" \
+                       "fv3jedi_convertstate.x" \
+                       "fv3jedi_eda.x" \
+                       "fv3jedi_forecast.x" \
+                       "fv3jedi_plot_field.x" \
+                       "fv3jedi_data_checker.py" \
+                       "fv3jedi_enshofx.x" \
+                       "fv3jedi_hofx_nomodel.x" \
+                       "fv3jedi_testdata_downloader.py" )
+            JEDI_EXE+=("soca_addincrement.x" \
+                        "soca_enspert.x" \
+                        "soca_hofx.x" \
+                        "soca_ensrecenter.x" \
+                        "soca_convertincrement.x" \
+                        "soca_ensvariance.x" \
+                        "soca_letkf.x" \
+                        "soca_convertstate.x" \
+                        "soca_error_covariance_training.x" \
+                        "soca_setcorscales.x" \
+                        "soca_diffstates.x" \
+                        "soca_forecast.x" \
+                        "soca_sqrtvertloc.x" \
+                        "soca_dirac.x" \
+                        "soca_gridgen.x" \
+                        "soca_staticbinit.x" \
+                        "soca_enshofx.x" \
+                        "soca_hofx3d.x" \
+                        "soca_var.x")
```

Comment:
Are all of these executables used in the workflow?

Reply:
@aerorahul , no, only a fraction are used currently but I'm also realizing that some of these will probably never be used. Should I minimize that list to what will eventually be used or to what's currently used?

Reply:
yes.  We should only link to what we are actively using.  There are other EE2 requirements for how they are built and the source code is linked, but I don't an idea how to address that issue. For e.g.`gsi.x` comes from a directory `gsi.fd`

Reply:
Yes this is something that we are going to have to work out with NCO, because it probably will be unreasonable to modify JEDI to put every single executable in its own folder when this is seemingly purely an aesthetic choice